W-17017644-reorg-permissions-duke (#311)
mulesoft/docs-monitoring: modules/ROOT/pages/am-permissions.adoc

//all monitoring permissions
include::access-management::partial$include-permissions-monitoring.adoc[tag=permsMonitoringAll]

> In addition to Monitoring Administrator and Anypoint Monitoring Viewer permissions,  asset-specific permissions are required to view and read API and app data for some deployments. For example, if you have the Anypoint Monitoring Viewer permission and want to view Anypoint Monitoring data on a hybrid Mule app deployment, you must also have the Read Applications permission (from Runtime Manager) in the same environment that the app is deployed to.

Author: ```suggestion
In addition to Monitoring Administrator and Anypoint Monitoring Viewer permissions, asset-specific permissions are required to view and read API and app data for some deployments. For example, if you have the Anypoint Monitoring Viewer permission and want to view Anypoint Monitoring data on a hybrid Mule app deployment, you must also have the Read Applications permission (from Runtime Manager) in the same environment that the app is deployed to.
```

//all monitoring permissions
include::access-management::partial$include-permissions-monitoring.adoc[tag=permsMonitoringAll]

> Access to data for users to with Anypoint Monitoring Administrator and Anypoint Monitoring Viewer permissions depends on other asset-specific permissions. To view or modify data related to specific apps and APIs, you also need the appropriate access permissions for those assets. For example, a user who has the Anypoint Monitoring Viewer permission and wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission (from Runtime Manager) in the same environment that the app is deployed to. 

Author: ```suggestion
To access to data, asset-specific permissions are required if you have Anypoint Monitoring Administrator or Anypoint Monitoring Viewer permissions. To view or modify data related to specific apps and APIs, you also need the appropriate access permissions for those assets. For example, a user who has the Anypoint Monitoring Viewer permission and wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission (from Runtime Manager) in the same environment that the app is deployed to. 
```
